examples/adafruit_io_simpletest_data.py

```diff
@@ -12,11 +12,11 @@
 # Import Adafruit IO REST Client
 from adafruit_io.adafruit_io import RESTClient, AdafruitIO_RequestError
 
-# Get wifi details and more from a wifi_settings.py.py file
+# Get wifi details and more from a esp32spi_secrets.py.py file
```

Comment:
also fix up all the .py.py references ... otherwise with the change ladyada requested - it looks good to me.

Reply:
Fixed the references